### PR TITLE
DOCSP-4384 Add analytics snippet to jazzy docs

### PR DESCRIPTION
**Pull request by cbush**
- As in other sdks, when the argument "analytics" is passed to
generate_docs.sh, then the analytics snippet will be added.
- Jazzy base theme "fullwidth" is copied to contrib/docs-theme/
and the analytics snippet is added there.
- Local builds of docs (i.e. without passing "analytics" argument)
will produce docs without analytics, using the standard fullwidth
theme.

Staged:
https://docs-mongodbcom-staging.corp.mongodb.com/stitch/bush/docsp-4383-analytics/sdk/swift/index.html

Note: to verify,
- view source and see analytics snippet in the header, something like
```    
    <script>
        !function(){var analytics=window.analytics=window.analytics ........
```
- go to https://app.segment.com/mongodb/sources/docs/debugger and see incoming analytics from staging.

### CODE REVIEW

File: contrib/docs-theme/fullwidth-analytics/assets/css/highlight.css.scss [hunk 1]
```diff
@@ -0,0 +1,63 @@
+/* Credit to https://gist.github.com/wataru420/2048287 */
```
Comment (cbush) [inline contrib/docs-theme/fullwidth-analytics/assets/css/highlight.css.scss]: This file copies from jazzy fullwidth theme

File: contrib/docs-theme/fullwidth-analytics/templates/analytics.mustache [hunk 2]
```diff
@@ -0,0 +1,9 @@
+<script>
+  !function(){var analytics=window.analytics=window.analytics||[];if(!analytics.initialize)if(analytics.invoked)window.console&&console.error&&console.error("Segment snippet included twice.");else{analytics.invoked=!0;analytics.methods=["trackSubmit","trackClick","trackLink","trackForm","pageview","identify","reset","group","track","ready","alias","debug","page","once","off","on"];analytics.factory=function(t){return function(){var e=Array.prototype.slice.call(arguments);e.unshift(t);analytics.push(e);return analytics}};for(var t=0;t<analytics.methods.length;t++){var e=analytics.methods[t];analytics[e]=analytics.factory(e)}analytics.load=function(t,e){var n=document.createElement("script");n.type="text/javascript";n.async=!0;n.src="https://cdn.segment.com/analytics.js/v1/"+t+"/analytics.min.js";var a=document.getElementsByTagName("script")[0];a.parentNode.insertBefore(n,a);analytics._loadOptions=e};analytics.SNIPPET_VERSION="4.1.0";
```
Comment (cbush) [inline contrib/docs-theme/fullwidth-analytics/templates/analytics.mustache]: SNIPPET_VERSION="4.1.0" is indeed the Segment snippet version, not an oversight on Stitch version.

**Comment by adamchel** [comment]
and evergreen is already broken until they install Xcode 10.2 so it should be okay to merge

**Comment by cbush** [comment]
Linting passed, evg otherwise broken until Xcode 10.2 installed. Merging.

**Pull request merged**